packages/bpk-component-spinner/changelog.yml

```diff
@@ -0,0 +1,55 @@
+upcoming:
+  changes:
+    minor: We are doing new things here
```

Comment:
I'm assuming this will go before merging?

Reply:
Eh yeah I'll comment it out